src/index.html

```diff
@@ -1,13 +1,420 @@
+        <h2 class="section-title explore-online__title">More to Explore Online</h2>
+        <div class="explore-online__content">
+          <article class="explore-online-block">
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-left.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Collection Areas</h3>
+              <p class="explore-online-block__description">
+                These collection areas study, exhibit, and care for the objects in The Met collection.
+              </p>
+            </div>
+          </article>
+          <article class="explore-online-block">
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-center.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Travel</h3>
+              <p class="explore-online-block__description">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </p>
+            </div>
+          </article>
+          <article class="explore-online-block">
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-right.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Virtual group tours</h3>
+              <p class="explore-online-block__description">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </p>
+            </div>
+          </article>
+        </div>
+      </section>
+
+      <section class="contact-us" id="contact-us">
+        <h2 class="section-title contact-us__title">Contact us</h2>
+        <div class="contact-us-content">
+          <div class="contact-us-content__fifth-avenue">
+            <h3
+              class="contact-us-content__title contact-us-content__text-stl-1">
+              The Met Fifth Avenue
+            </h3>
+            <p
+              class="contact-us-content__phone contact-us-content__text-stl-2"
+            >
+              Phone
+            </p>
+            <a
+              href="tel:+12125357710"
+              class="contact-us-content__phone-number contact-us-content__text-stl-1">
```

Comment:
```suggestion
            <a href="tel:+12125357710" class="contact-us-content__phone-number contact-us-content__text-stl-1">
```

Reply:
can we wrap the same way a long links like this one?
href="https://www.google.com/maps/place/1000+5th+Ave,+New+York,+NY+10028,+USA/@40.7794355,-73.9633844,17z/data=!3m1!4b1!4m6!3m5!1s0x89c25896dcda965d:0x5bbcfab18ea2c739!8m2!3d40.7794355!4d-73.9633844!16s%2Fg%2F1yl46zflv?entry=ttu"